package.json

```diff
@@ -603,70 +593,66 @@
     "value": "#131F37"
   },
   {
-    "name": "pPoloniexMaidsafe",
-    "title": "Poloniex Bitcoin/Maidsafe (\u0243/MAID) ticker?",
+    "name": "pKrakenEther",
+    "title": "Kraken Bitcoin/Ether (\u0243/Ether) ticker?",
     "type": "bool",
     "value": false
   },{
-    "name": "pPoloniexMaidsafeColor",
-    "title": "Poloniex Bitcoin/Maidsafe Ticker color",
+    "name": "pKrakenEtherColor",
+    "title": "Kraken Bitcoin/Ether Ticker color",
     "type": "color",
-    "value": "#000000"
+    "value": "#FFF"
   },
   {
-    "name": "pPoloniexBitcoindark",
-    "title": "Poloniex Bitcoin/BitcoinDark (\u0243/BTCD) ticker?",
+    "name": "pKrakenEtherEUR",
+    "title": "Kraken EUR/Ether (\u20ac/ETH) ticker?",
     "type": "bool",
     "value": false
   },{
-    "name": "pPoloniexBitcoindarkColor",
-    "title": "Poloniex Bitcoin/BitcoinDark Ticker color",
+    "name": "pKrakenEtherEURColor",
+    "title": "Kraken EUR/Ether Ticker color",
     "type": "color",
-    "value": "#000000"
+    "value": "#FFF"
   },
   {
-    "name": "pPoloniexMonero",
-    "title": "Poloniex Bitcoin/Monero (\u0243/XMR) ticker?",
+    "name": "pKrakenEtherUSD",
+    "title": "Kraken USD/Ether ($/ETH) ticker?",
     "type": "bool",
     "value": false
   },{
-    "name": "pPoloniexMoneroColor",
-    "title": "Poloniex Bitcoin/Monero Ticker color",
+    "name": "pKrakenEtherUSDColor",
+    "title": "Kraken USD/Ether Ticker color",
     "type": "color",
-    "value": "#413ADF"
+    "value": "#FFF"
   },
   {
-    "name": "pPoloniexMoneroUSD",
-    "title": "Poloniex USD/Monero ($/XMR) ticker?",
+    "name": "pPoloniexBelaCoin",
```

Comment:
All the new coins introduced don't have color setting, that will make using multiple coins with the same default color confusing because users can't change the color.
Can you please add the color variable with some default value for all the new coins you introduced?